llvm/tools/repo2obj/R2OELFSymbolTable.h

```diff
@@ -316,10 +316,12 @@ auto SymbolTable<ELFT>::sort() -> std::vector<Value *> {
   std::stable_sort(
       std::begin(OrderedSymbols), std::end(OrderedSymbols),
       [](Value const *const A, Value const *const B) {
-        return (A->linkage() == pstore::repo::linkage_type::internal ||
-                A->linkage() == pstore::repo::linkage_type::internal_no_symbol) &&
-               (B->linkage() != pstore::repo::linkage_type::internal &&
-                B->linkage() != pstore::repo::linkage_type::internal_no_symbol);
+        auto ALinkage = A->linkage();
+        auto BLinkage = B->linkage();
```

Comment:
Both ALinkage and BLinkage could be const.

Reply:
Thanks, Fixed it.